frontend/src/libs/components/icon/common.ts

```diff
@@ -1,3 +1,4 @@
+import { ReactComponent as Arrow } from '~/assets/img/icons/arrow.svg';
```

Comment:
I believe `ArrowDown` is the same thing - if so then we don't need a duplicate

Reply:
Not exactly. My svg contains only arrow but arrowDown contains arrow and white space around it.
I guess I can replace arrowDown with my arrow then.

Reply:
Yep 👍